src/Workspaces/Core/Portable/PatternMatching/PatternMatcher.cs

```diff
@@ -52,7 +53,14 @@ internal abstract partial class PatternMatcher : IDisposable
             bool allowFuzzyMatching = false)
         {
             culture = culture ?? CultureInfo.CurrentCulture;
+            if (culture.Name == "en-US")
+            {
+                culture = CultureInfo.InvariantCulture;
```

Comment:
:question: If the behavior is different, how can we be sure it's not affecting the quality of the results?